apps/editing-toolkit/editing-toolkit-plugin/wpcom-block-editor-nav-sidebar/src/attach-sidebar.tsx

```diff
@@ -34,7 +35,7 @@ if ( typeof MainDashboardButton !== 'undefined' ) {
 				// eslint-disable-next-line react-hooks/exhaustive-deps
 			}, [] );
 
-			const showLaunchpad = getQueryArg( window.location.search, 'showLaunchpad' );
+			const isStartWritingFlow = getQueryArg( window.location.search, START_WRITING_FLOW );
```

Comment:
[nit] It might be better to check whether the value is true here as well

Reply:
Changed it [here](https://github.com/Automattic/wp-calypso/pull/76082/commits/9c5c54295e647f264e6c126191150f42127f4ae2) 🙏